helm-chart/mezod/README.md

```diff
@@ -0,0 +1,61 @@
+# Helm chart for Mezod
+
+The chart contains configuration for testnet on Sepolia so far.
+Eventually, it will be updated to support mainnet by default.
+
+# mezod
+
+![Version: 0.0.1](https://img.shields.io/badge/Version-0.0.1-informational?style=flat-square) ![AppVersion: v0.1.0-rc0](https://img.shields.io/badge/AppVersion-v0.1.0--rc0-informational?style=flat-square)
```

Comment:
The latest version is `v0.1.0-rc1`. Also, do we need to update it automatically on every release or is there a way to automate this badge?